neonvm/runner/main.go

```diff
@@ -68,6 +68,9 @@ const (
 	// in microseconds. Min 1000 microseconds, max 1 second
 	cgroupPeriod     = uint64(100000)
 	cgroupMountPoint = "/sys/fs/cgroup"
+
+	// Estimated time that the VM startup takes. The CPU limit is set only after this.
+	cpuThrottleDelay = 10 * time.Second
```

Comment:
Nit: IME boot takes ~1s; if that's the case, I'm hesitant to make this >5s (because longer durations will have a greater chance of being user-visible). Have you observed something different?